### ISSUE TITLE
Display data is now required

### ISSUE THREAD

**Issue by meganrm** [simularium/simularium-viewer#282]
## What needs to happen?
No longer default to SPHERE, error out if no display data

## Why should we do this?
Simulariumio has been updated to have SPHERE GROUP which means we no longer have a default display_type

## When does this need to get done?
Once the change is merged into simulariumio and a new version is released

**Comment by blairlyons** [comment]
this has been released in simulariumio

**Comment by toloudis** [comment]
We used to use sphere as a proxy geometry in case agent data arrived and was ready before the geometry arrived.

**Comment by meganrm** [comment]
Yeah, I'm looking at this now, and we do give a warning with there is no display type given before we default it to a sphere. I know I'm the one that wrote this, but I'm now not sure why we should do this. Ie, alerting the user that they're missing data, but still rendering SOMETHING seems to be a good behavior?

**Comment by toloudis** [comment]
It seems like if display data is required data, (on the simulariumio side and here) then we're good to go!  We don't need to get rid of the sphere default, unless you really don't want to see agents whose geometry hasn't arrived yet or has errored out..

**Comment by meganrm** [comment]
Yeah, so @blairlyons is it ok for me to not do anything here? We've already implemented SPHERE GROUP in the viewer. and again, we do warn users of their mistake if the data is missing. 
```TypeScript 
export const makeMissingDisplayTypeErrorMessage = (
    key: string,
    url: string
): string => {
    if (url) {
        return `Missing typeMapping[${key}].geometry.displayType, so we couldn't request ${url}. Geometry will default to spheres`;
    } else {
        return `No typeMapping[${key}].geometry.displayType. Geometry will default to spheres`;
    }
};
```
